frontend/src/modules/Shared/Shares/ShareEditForm.js

```diff
@@ -268,6 +268,7 @@ export const ShareEditForm = (props) => {
         },
         variant: 'success'
       });
+      navigate(`/console/shares/${share.shareUri}`);
```

Comment:
On line 278 onApply function is called. 
If we redirect before it's called, than we break the logic.
Currently there is nothing very important there, but later we may want to add some logic in onApply.

Reply:
@SofiaSazonova what should we do instead?

Reply:
@SofiaSazonova Are you suggesting I add `navigate(`/console/shares/${share.shareUri}`);` on line 279 after `onApply();` instead of on line 271?

Reply:
@anushka-singh - yes I think just move the `navigate(...)` after calling `onApply();`

Also I imagine this works when submitting share from console - but what about when editing a share from `ShareView` form (we re-use the same `ShareEditForm`)

Maybe we can use `const location = useLocation();` and only redirect when not already on ShareView?

Reply:
Yes, conditional redirect is a good idea.

Reply:
Added conditional redirect. 
@noah-paige @SofiaSazonova Please take a look.